stretch-assignment/digital_timer/index.js

```diff
@@ -0,0 +1,103 @@
+(function() {
+  let startButton = document.createElement("button");
+  let resetButton = document.createElement("button");
+  let brEl = document.createElement("br");
+  startButton.textContent = "Start 10 second Timer";
+  startButton.setAttribute("id","start");
+  resetButton.setAttribute("id","reset");
+  startButton.setAttribute("onClick","javascript:startTimer()");
+  resetButton.setAttribute("onClick","javascript:resetTimer()");
+  resetButton.textContent = "Reset Timer";
+  document.querySelector(".digits").append(brEl);
+  document.querySelector(".digits").append(startButton);
+  document.querySelector(".digits").append(resetButton);
+})();
```

Comment:
i'm curious to know why you chose to wrap this inside an immediately invoked anonymous function.
this code block would run the same way if it were placed in the global scope.

Reply:
Good to know